TimeService.test.ts

```diff
@@ -0,0 +1,21 @@
+import {TimeService} from "./TimeService";
+
+describe('TimeService', () => {
+    it('can test getCurrentTimeString', () => {
+        const now = new Date();
+        const currentTimeString = now.toISOString();
+
+        expect(TimeService.getCurrentTimeString()).toBeTruthy();
+        expect(parseInt(TimeService.getCurrentTimeString())).toBeGreaterThanOrEqual(parseInt(currentTimeString));
+        expect.assertions(2)
+
+    });
+
+    it('can test getTimeAfterMonths', () => {
+        const now = new Date();
+        const currentTimeString = now.toISOString();
+
+        expect(TimeService.getTimeAfterMonths(currentTimeString, 12)).toBeTruthy();
+
+    });
+});
```

Comment:
Generally it is a very good idea to end every file with a newline, since for example, some programs may handle the end of file in an unexpected way if it is not a plain newline (many IDEs have an option to ensure that automagically).